llvm/include/llvm/IR/IntrinsicsAMDGPU.td

```diff
@@ -2359,6 +2359,12 @@ def int_amdgcn_mbcnt_hi :
   DefaultAttrsIntrinsic<[llvm_i32_ty], [llvm_i32_ty, llvm_i32_ty],
             [IntrNoMem]>;
 
+def int_amdgcn_bcnt32_lo :
```

Comment:
This does not need an intrinsic. This doesn't help match the pattern in any cases, and introduces new support burdens to every optimization